src/io/sb3/toSb3.ts

```diff
@@ -1,163 +1,1114 @@
-const doesThisCodeHaveErrors = "not if I comment it out";
-export default doesThisCodeHaveErrors;
+import Block, { BlockBase, KnownBlock, ProcedureBlock } from "../../Block";
+import Costume from "../../Costume";
+import Project, { TextToSpeechLanguage } from "../../Project";
+import Script from "../../Script";
+import Sound from "../../Sound";
```

Comment:
It seems like `KnownBlock`, `ProcedureBlock`, `Costume`, `Script`, and `Sound` are all unused imports.
```suggestion
import Block, { BlockBase } from "../../Block";
import Project, { TextToSpeechLanguage } from "../../Project";
```